pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -38,13 +38,16 @@ import (
 	v1 "kubevirt.io/client-go/api/v1"
 	"kubevirt.io/client-go/log"
 	"kubevirt.io/client-go/precond"
+	migrationproxy "kubevirt.io/kubevirt/pkg/virt-handler/migration-proxy"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/converter"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/network/cache"
 )
 
 var bridgeFakeIP = "169.254.75.1%d/32"
 
+const LibvirtLocalConnectionPort = 22222
```

Comment:
Can this const port number can be imported from [here](https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-launcher/virtwrap/manager.go#L74)?

Reply:
Unfortunately no, because that would create cycle in deps

Reply:
och, never mind then. Thanks.